README.md

```diff
@@ -1,7 +1,5 @@
 # Developer notes
 
-[![Pocket Marketing Pages](https://github.com/mozilla-l10n/mozl10n-linter/actions/workflows/pocket.yaml/badge.svg)](https://github.com/mozilla-l10n/mozl10n-linter/actions/workflows/pocket.yaml)
```

Comment:
@peiying2 Not sure why this has disappeared in this PR but hopefully we can bring it back manually. The new string below needs to flow to Smartling ASAP